app/src/androidTest/resources/reference_tests/domain_matching_tests.json

```diff
@@ -77,10 +77,7 @@
         "siteURL": "https://randomsite123.com/",
         "requestURL": "https://bad.cnames.test/something",
         "requestType": "script",
-        "expectAction": "block",
-        "exceptPlatforms": [
-          "android-browser"
-        ]
+        "expectAction": "block"
```

Comment:
PR for this change here: https://github.com/duckduckgo/privacy-reference-tests/pull/56